Bug 1902584: note workaround for advanced networking issues (#28036)
openshift/openshift-docs: modules/installation-user-infra-machines-static-network.adoc

bond=bond0:em1,em2:mode=active-backup
ip=10.10.10.2::10.10.10.254:255.255.255.0:core0.example.com:bond0:none
----

[IMPORTANT]
====

> When using the advanced networking options, you may encounter issues during the first boot of {op-system} where the statically configured address is not present or not activated properly. In such cases, you may need to manually reboot the {op-system} machine in order to workaround this problem. In newer versions of {op-system}, the network code was significantly improved and this issue was resolved.  See link:https://bugzilla.redhat.com/show_bug.cgi?id=1902584[RHBZ#1902584] for additional details.

Reviewer: ```suggestion
When using the advanced networking options, you could encounter issues during the first boot of {op-system} where the statically configured address is not present or not activated properly. In such cases, you might need to manually reboot the {op-system} machine to workaround this problem. In newer versions of {op-system}, this issue is resolved. See link:https://bugzilla.redhat.com/show_bug.cgi?id=1902584[BZ#1902584] for additional details.
```
